keyboards/helixlab/aaru65/hotswap/keymaps/default/keymap.c

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 luantty2 (@luantty2)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+#ifdef FADER_ENABLE
+#    include "fader.h"
+#endif
+#include "debug_list.h"
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names { _BASE, _FN };
```

Comment:
A comma after the last entry will prevent it from being reformatted to one line. 
```suggestion
enum layer_names { 
    _BASE, 
    _FN,
};
```